src/Database/LibSQLConnection.php

```diff
@@ -54,14 +55,6 @@ public function useDefaultSchemaGrammar()
         }
     }
 
-    public function createReadPdo(array $config): ?LibSQLDatabase
-    {
-        $db = new LibSQLDatabase($config);
-        $this->setReadPdo($db);
-
-        return $db;
-    }
-
```

Comment:
This function should not be removed, considering that **libSQL** is _not built under PDO_. This will cause problems when running artisan commands related to the database.

Reply:
I'll look into it, can rollback. 
Removed since it was not used anymore.
The only place it's used is \Illuminate\Database\Connectors\ConnectionFactory::make which is called when DBManager creates connection via ConnectionFactory, but in our case we extend it, so it will not use ConnectionFactory.
But yea it's easy to rollback. Will look today

Reply:
Looking more closely, createReadPdo() is not related in any way to connection, it's actually ConnectionFactory method, which is not used by this driver. So i don't see why should we have this method.